code/CloudFlare_Purge.php

```diff
@@ -121,26 +123,35 @@ private function rglob($pattern, $flags = 0)
     {
         $files = glob($pattern, $flags);
         foreach (glob(dirname($pattern) . '/*', GLOB_ONLYDIR | GLOB_NOSORT) as $dir) {
+            if (basename($dir) == 'framework') {
+                continue;
+            }
+            // if we implode the array we should be able to search for all extensions at once like below
```

Comment:
This is definitely a priority, the more extensions to search for the longer the request takes.

Windows 10 WAMP PHP 5.5 - Unable to get any results from the GLOB_BRACE approach